src/meshes/p4est_mesh.jl

```diff
@@ -469,12 +474,186 @@ function p4est_mesh_from_standard_abaqus(meshfile, mapping, polydeg,
+                        boundary_names[1, tree] = boundary
+                    end
+                    # Check "right face" (x_max)
+                    if tree_nodes[2] in node_set_dict[boundary] &&
+                       tree_nodes[3] in node_set_dict[boundary] &&
+                       tree_nodes[6] in node_set_dict[boundary] &&
+                       tree_nodes[7] in node_set_dict[boundary]
+                        # Right face is position 2 in p4est indexing
+                        boundary_names[2, tree] = boundary
+                    end
+                    # Check "bottom face" (z_min)
+                    if tree_nodes[1] in node_set_dict[boundary] &&
+                       tree_nodes[2] in node_set_dict[boundary] &&
+                       tree_nodes[3] in node_set_dict[boundary] &&
+                       tree_nodes[4] in node_set_dict[boundary]
+                        # Bottom face is position 5 in p4est indexing
+                        boundary_names[5, tree] = boundary
+                    end
+                    # Check "top face" (z_max)
+                    if tree_nodes[5] in node_set_dict[boundary] &&
+                       tree_nodes[6] in node_set_dict[boundary] &&
+                       tree_nodes[7] in node_set_dict[boundary] &&
+                       tree_nodes[8] in node_set_dict[boundary]
+                        # Top face is position 6 in p4est indexing
+                        boundary_names[6, tree] = boundary
+                    end
+                end
+            end
+        end
+    end
 
     return p4est, tree_node_coordinates, nodes, boundary_names
 end
 
+function parse_elements(meshfile, n_trees, n_dims)
+    @assert n_dims ∈ [2, 3] "Only 2D and 3D meshes are supported"
+    element_types = n_dims == 2 ?
+                    [
+        "*ELEMENT, type=CPS4",
+        "*ELEMENT, type=C2D4",
+        "*ELEMENT, type=S4",
+    ] :
+                    ["*ELEMENT, type=C3D8"]
+    # 2D quads: 4 nodes + element index, 3D hexes: 8 nodes + element index                                                               
+    expected_content_length = n_dims == 2 ? 5 : 9
+
+    element_node_matrix = Matrix{Int64}(undef, n_trees, expected_content_length - 1)
+    el_list_follows = false
+    tree_id = 1
+
+    open(meshfile, "r") do file
+        for line in eachline(file)
+            if any(startswith(line, el_type) for el_type in element_types)
+                el_list_follows = true
+            elseif el_list_follows
+                content = split(line, ",")
+                if length(content) == expected_content_length # Check that we still read in connectivity data
+                    content_int = parse.(Int64, content)
+                    # Add constituent nodes to the element_node_matrix
+                    element_node_matrix[tree_id, :] = content_int[2:end] # First entry is element id
+                    tree_id += 1
+                else # Read all elements for this ELSET
```

Comment:
```suggestion
                else # Read all elements for this ELSE
```

Reply:
This is actually no typo, but an abbreviation for **El**ement**Set** see https://gist.github.com/DanielDoehring/eefe73ae5d113bc3944a518b6e88e663#file-cube_boundaries-inp-L292

Reply:
Whoops, I missed this. Sorry